ComputerCraft.tmLanguage

```diff
@@ -4,7 +4,7 @@
 <plist version="1.0">
 <dict>
 	<key>comment</key>
-	<string>ComputerCraft Syntax: Version 1.2</string>
+	<string>ComputerCraft Syntax: Version 1.0.4</string>
```

Comment:
Why did you change the version number? Incrementing it to 1.3 would've been logical, but 1.0.4 has no relation to anything.